.github/pull_request_template.md

```diff
@@ -0,0 +1,21 @@
+## Describe the problem
```

Comment:
It might be worth putting placeholder text here to guide people, eg
```
<!-- Write 2-3 sentences summarising the reasons behind this change, this might be the problem you're solving, or it could be the context around the request !-->
```

I'd also throw in the mix for consideration folding the first two into one heading and having the comment ask for
- Describe the problem
- Describe the change
- Explain why you're making the change

Reply:
Makes sense. Combined the first 2 sections and added a placeholder text - see [a899604](https://github.com/wiredtiger/wiredtiger/pull/10973/commits/a89960439bb3b16fcec29a406a651a2dcf37a2db)

Reply:
I agree that might work better.